### ISSUE TITLE
[pni] homepage: un-nest figcaption

### ISSUE THREAD

**Issue by Pomax** [mozilla/foundation.mozilla.org#4642]
The `<figcaption>` that we use in `<figure>` elements for each product cannot be nested inside another element and needs to be a direct child of `<figure>`.

![](https://user-images.githubusercontent.com/177243/81863466-20341e00-9520-11ea-82f9-304afd800e61.png)
